test/TensorFlowRuntime/string.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+//
+// String Tensor tests.
+
+import TensorFlow
+import StdlibUnittest
+
+var StringTensorTests = TestSuite("String")
+
+StringTensorTests.test("StringComparison") {
+
+  let t1 = Tensor("foo")
+  let result1 = t1.elementsEqual(t1)
+  expectEqual(ShapedArray(shape: [], scalars: [true]), result1.array)
+
+  let t2 = Tensor(["foo", "bar"])
+  let result2 = t2.elementsEqual(t2)
+  expectEqual(ShapedArray(shape: [2], scalars: [true, true]),
+              result2.array)
+}
```

Comment:
also add a test where two string tensors are not equal? this can help confirm we are passing swift string literals properly into TF tensors.

Reply:
Done